resources/js/functions.js

```diff
@@ -392,15 +392,55 @@ function getOS() {
   return osname;
 }
 
+function toggleElement(evt, elemId) {
+  var btn = evt.target;
+  var elem = document.getElementById(elemId);
+  if (elem.style.display === "none") {
+    elem.style.display = "block";
+  } else {
+    elem.style.display = "none";
+  }
+  $(btn).toggleClass("alt-details-closed");
+}
+
+$(document).ready(function() {
+  $('.alt-details-toggle').click();
```

Comment:
Would it be possible to remove this line? Change the CSS so that by default this element is correctly displayed (or hidden).

Reply:
I understand now that this is needed to handle visitors with disabled JS